webviz_subsurface/_datainput/from_timeseries_cumulatives.py

```diff
@@ -0,0 +1,157 @@
+from typing import List, Union
+
+import pandas as pd
+import numpy as np
+
+# pylint: disable=dangerous-default-value
+def calc_from_cumulatives(
+    data: pd.DataFrame,
+    column_keys: Union[List[str], str],
+    time_index: str,
+    time_index_input: str,
+    as_rate: bool,
+) -> pd.DataFrame:
+    """Calculates interval delta and average rate at given time interval `time_index`.
+    Assumes that the data is already sampled to a time interval `time_index_input`.
+    `time_index` can currently be `daily`, `monthly` or `yearly`.
+    Interpolation is not perforamed, and the time interval `time_index` therefore has to be shorter
+    or equal to `time_index_input`, (e.g. is `time_index` = `yearly` ok with
+    `time_index_input` = `monthly`, but the opposite is invalid)
+    `column_keys` define vectors to take calculate for of.
```

Comment:
Think there should be no comma in front of the parenthesis on line 18, and a period after the parenthesis closes on line 19. Then line 20 is a separate sentence.

Agree that it is not the easiest thing to read. Will add some blank lines instead of just linebreaks to make subsections clearer.